masonry/src/widgets/grid.rs

```diff
@@ -246,7 +264,11 @@ impl Grid {
         this.ctx.request_layout();
     }
 
-    #[expect(missing_docs, reason = "TODO")]
+    /// Removes a child widget at the specified index.
```

Comment:
Sometimes we use "specified" and sometimes "given" ... should decide on one (everywhere).